### ISSUE TITLE
Move Value for Sync Offset?

### ISSUE THREAD

**Issue by DachsbauTV** [exeldro/obs-move-transition#102]
Hi @exeldro 

Thanks for the amazing plugin! It's been a great help on so many different levels, we didn't even thought of before.

One little thing: I am trying to create quick controls to adjust audio delay during live with the sync offset function in the advanced audio controls. 

Would it be possible to add this value to your "Move Value" filter somehow?

**Comment by exeldro** [comment]
Like using a move value filter to change the value of a delay filter?

**Comment by DachsbauTV** [comment]
Yes but unfortunately there is no audio delay filter in OBS or available as a plugin, I guess because most people rely on this feature, which already exists in OBS:
https://streamshark.io/obs-guide/audio-delay
